core/modules/user/css/user.css

```diff
@@ -14,26 +14,17 @@
 #permissions tr.even .form-item {
   white-space: normal;
 }
+
 #user-admin-settings fieldset .fieldset-description {
   font-size: 0.85em;
   padding-bottom: .5em;
 }
 
-/**
- * Override default textfield float to put the "Add role" button next to
- * the input textfield.
- */
-#user-admin-roles td.edit-name {
-  clear: both;
+#user-roles .system-role {
+  color: #666;
 }
-#user-admin-roles .form-item-name {
-  float: left; /* LTR */
-  margin-right: 1em; /* LTR */
-}
-[dir="rtl"] #user-admin-roles .form-item-name {
-  float: right;
-  margin-left: 1em;
-  margin-right: 0;
+#user-roles .description {
+  margin-left: 2em;
 }
```

Comment:
This entire section of CSS is moot, as `theme_user_admin_roles()` as it is currently implemented renders the text field and the "Add role" button into separate cells/columns in the table.

Reply:
Filed a follow-up: https://github.com/backdrop/backdrop-issues/issues/4506